docs/source/extension/extension_migration.rst

```diff
@@ -32,6 +32,19 @@ breaking change when defining fixture values that are array of objects.
 
 See the `Playwright 1.46.0 release notes <https://github.com/microsoft/playwright/releases/tag/v1.46.0>`_ for more information.
 
+
+Shared model
+~~~~~~~~~~~~
+
+The outputs set on the shared cell model are now expected to be wrapped
+in the pycrdt ``Map`` objects rather than provided as plain dictionaries.
```

Comment:
```suggestion
in the pycrdt ``Map`` objects rather than provided as plain dictionaries.
Further, the `"text"` entry can now be specified as an `Array<string>`
object for outputs of `"stream"` type, allowing for better performance.
```